<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="datasheet_tab">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <layout class="QGridLayout" name="tab_item_base">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_file_new"/>
    <addaction name="separator"/>
    <addaction name="action_file_load"/>
    <addaction name="action_file_save"/>
    <addaction name="action_file_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_file_quit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>도구</string>
    </property>
    <addaction name="actionRun_Scripts"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>분석</string>
    </property>
    <widget class="QMenu" name="menuDescriptive_Statistics">
     <property name="title">
      <string>Descriptive Statistics</string>
     </property>
     <addaction name="actionFrequency_Analysis"/>
    </widget>
    <addaction name="menuDescriptive_Statistics"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_file_new">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRun_Scripts">
   <property name="text">
    <string>Run Script</string>
   </property>
  </action>
  <action name="action_file_save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="action_file_load">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFrequency_Analysis">
   <property name="text">
    <string>Frequency Analysis</string>
   </property>
  </action>
  <action name="action_file_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
